modules/oe_webtools_social_share/tests/FunctionalJavascript/SocialShareSettingsFormTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_webtools_social_share\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Tests the Social share widget settings form.
+ */
+class SocialShareSettingsFormTest extends WebDriverTestBase {
```

Comment:
There is no reason to have this a Javascript test, this can be simply a Functional test,